lib/prompts/createProjectPrompt.ts

```diff
@@ -42,14 +54,23 @@ function findTemplateByNameOrLabel(
   );
 }
 
+// Includes `projectTemplate` in the return value if `projectTemplates` is provided
+export async function createProjectPrompt(
+  promptOptions: { name?: string; dest?: string; template?: string },
```

Comment:
Nit: this inline type could be moved to a type and reused

Reply:
I like that idea, done in [df00fca](https://github.com/HubSpot/hubspot-cli/pull/1449/commits/df00fca88f867c909716a04776e8b2df7327025b)